README.md

```diff
@@ -174,19 +169,7 @@ We highly encourage running and updating the tests to make sure no regressions h
 
     make check
 
-You can also build a test coverage report. This requires that the packages `lcov`, `gcov` and `genhtml` be installed. On Ubuntu you can get these with:
-
-    sudo apt-get install lcov
-
-To make the coverage report, configure the build for it:
-
-    ./configure --enable-coverage
```

Comment:
It is also possible to make local code coverage reports
`cmake .. -DENABLE_COVERAGE=On`

and `make coverage` below